Modules/Setup.stdlib.in

```diff
@@ -78,7 +78,7 @@
 # hashing builtins, can be disabled with --without-builtin-hashlib-hashes
 @MODULE__MD5_TRUE@_md5 md5module.c
 @MODULE__SHA1_TRUE@_sha1 sha1module.c
-@MODULE__SHA256_TRUE@_sha256 sha256module.c
+@MODULE__SHA256_TRUE@_sha256 _hacl/sha256module.c _hacl/Hacl_Streaming_SHA2.c -I$(srcdir)/Modules/_hacl/include -I$(srcdir)/Modules/_hacl/internal -I$(srcdir)/Modules/_hacl/krmllib/dist/minimal -DKRML_VERIFIED_UINT128 -D_BSD_SOURCE -D_DEFAULT_SOURCE
```

Comment:
Includes and defines belong into `PY_STDLIB_MOD` block for sha256module in `configure.ac`.

Reply:
Thanks for pointing me to the right location... I've fixed this, regenerated the configure script using autoconf-2.69 (to minimize the diff), and also moved sha256module.c (and the clinic-generated file) back into their original locations.